client/components/bloganuary-header/index.tsx

```diff
@@ -0,0 +1,39 @@
+import { useTranslate } from 'i18n-calypso';
+import { useDispatch } from 'react-redux';
+import BloganuaryIcon from 'calypso/components/blogging-prompt-card/bloganuary-icon';
+import isBloganuary from 'calypso/data/blogging-prompt/is-bloganuary';
+import { recordTracksEvent } from 'calypso/state/analytics/actions';
+import './style.scss';
+
+const BloganuaryHeader = () => {
+	const dispatch = useDispatch();
+	const translate = useTranslate();
+
+	const trackBloganuaryMoreInfoClick = () => {
+		dispatch( recordTracksEvent( 'reader_bloganuary_more_info_click' ) );
+	};
+
```

Comment:
We could benefit from a `viewed` type event to be able to understand user interaction banner. ex. `calypso_reader_bloganuary_banner_viewed` being sent on mount.

One option is putting this in a useEffect on mount and also checking the `isBloganuary` condition to see if we should fire the event. Although it feels a little weird checking conditions that represent it rendering the component vs. it actually rendering the component. So another thought would be to use the useEffect on mount, but also add an outer wrapper component to this one so that it never renders outside of Bloganuary:


```
export default const BloganuaryHeaderWrapper = () => {
   if (! isBloganuary() ) return null;

  return <BloganuaryHeader />
}
```

And then we wouldn't have to rely on a check for `isBloganuary` in the <BloganuaryHeader />'s useEffect for determining whether or not to send the viewed event since render would imply viewed in that case.